AngularBFF/AngularBFF.Net8.Api/appsettings.Development.json

```diff
@@ -1,14 +1,15 @@
 {
   "Logging": {
     "LogLevel": {
-      "Default": "Information",
-      "Microsoft.AspNetCore": "Warning"
+      "Default": "Trace",
+      "Microsoft.AspNetCore": "Trace"
```

Comment:
Should we set these back to Information?

Reply:
I like to have it to trace for development. But can set it back and rather use the secret file that I have added now.

Reply:
.. and warning.